packages/docs/docs/getting-started.md

```diff
@@ -60,4 +60,4 @@ $ yarn workspace create-react-app-demo start
 ## Next Steps
 
 Now that you have some basic examples working, learn more about AI.JSX development
-by following the [tutorial](./tutorials/part1-completion.md).
+by following the [tutorial](./tutorials/aijsxTutorials/part1-completion.md).
```

Comment:
oh man i thought Docusaurus was going to alert me of broken links when I made the change in the folder structure. i will go through all the links to make sure things are working fine.